apis/cni/v1alpha1/calicoconfig_types.go

```diff
@@ -34,26 +23,31 @@ type Calico struct {
 }
 
 type Config struct {
-	// Maximum transmission unit setting
+	// Maximum transmission unit setting. "0" as default means MTU will be auto detected
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:default:=0
-	// "0" as default means MTU will be auto detected
 	VethMTU int64 `json:"vethMTU,omitempty"`
 }
 
 // CalicoConfigStatus defines the observed state of CalicoConfig
-type CalicoConfigStatus struct{}
+type CalicoConfigStatus struct {
+	SecretName string `json:"secretName,omitempty"`
```

Comment:
I think we use SecretRef as the convention for linking to a name. Can we use that instead?

Reply:
So ```SecretRef``` as a name without pointing to an object? And being a ```name``` instead?

Reply:
Yeah just SecretRef string that is the name. Could you make sure the other Config APIs are using the same pattern?

```SecretRef string `json:"secretRef,omitempty"```